File: 3515-FindIfDigitGameCanBeWon/3515-FindIfDigitGameCanBeWon.js
// Last updated: 3/6/2026, 2:08:16 PM
/**
 * @param {number[]} nums
 * @return {boolean}
 */
var canAliceWin = function(nums) {
    let singleSum = 0;
    let doubleSum = 0;
    for(let i=0;i<nums.length;i++){
        if(nums[i]<=9){
            singleSum+=nums[i];
        }
        else{
            doubleSum+=nums[i];
        }
    }
    return singleSum != doubleSum;
};

File: 3515-FindIfDigitGameCanBeWon/3515-FindIfDigitGameCanBeWon.cpp
// Last updated: 7/17/2026, 4:23:48 PM
class Solution {
public:
    bool canAliceWin(vector<int>& nums) {
        int singleSum = 0;
        int doubleSum = 0;

        for (int i = 0; i < nums.size(); i++) {
            if (nums[i] <= 9) {
                singleSum += nums[i];
            } else {
                doubleSum += nums[i];
            }
        }

        return singleSum != doubleSum;
    }
};